cmake_modules/FindEXAMPLE_PLUGIN_DEP.cmake

```diff
@@ -46,7 +46,7 @@ IF (NOT EXAMPLE_PLUGIN_DEP_FOUND)
   #"#define EXAMPLE_PLUGIN_DEP_PATCH 2"
   #The minimum version to requirement is made visible to this cmake file from
   #HPCC/plugins/exampleplugin/CMakeLists.txt:ADD_PLUGIN(EXAMPLEPLUGIN PACKAGES EXAMPLE_PLUGIN_DEP MINVERSION 4.6.2)
-  IF  (EXAMPLE_PLUGIN_DEP_INCLUDE_DIR)
+  IF  (EXISTS "${EXAMPLE_PLUGIN_DEP_INCLUDE_DIR}/hpcc-example-plugin-deps.h")
```

Comment:
Changed this line to reference the hpcc-example-plugin-deps.h file, similar to the other EXIST calls.